config/components/config/holiday_table.html

```diff
@@ -0,0 +1,31 @@
+{% component 'tile/tile' root_class_tags="grow max-w-screen lg:basis-2/5  lg:flex-none" %}
+    {% fill 'title' %}Holidays{% endfill %}
+    {% fill 'content' %}
+        <table class="table table-striped" style="width: auto;">
```

Comment:
I thought it's table-zebra in tailwind/daisy? Also, I think width: auto is no longer required.